### DISCUSSION TITLE
Netbox on raspberry pi4 via docker-compose

### DISCUSSION THREAD

**Discussion by Scott8586** [netbox-community/netbox-docker#523] [Getting Help] [answered]
I have been trying to install Netbox on a raspberry pi4  (4GB) running buster.  I've updated to the requisite versions of docker and docker-compose, but I'm getting an error on startup.  I think the primary problem is:

```
netbox-worker_1  | standard_init_linux.go:219: exec user process caused: exec format error
netbox_1         | standard_init_linux.go:219: exec user process caused: exec format error
```

I understand this may be when your host machine has a different architecture from your guest container image, but I don't know what to do about it.

Anyone have Netbox running via docker-composer on a pi4/buster?  Any hints welcome...

**Comment by cimnine** [comment, marked as answer]
The Raspberry PI runs on a chip with an ARM architecture, but currently NetBox Docker does not provide a Docker image for the ARM chip architecture.

**Reply by cimnine** [reply]
See https://github.com/netbox-community/netbox-docker/issues/468

**Reply by tribp** [reply]
Hi,  after some sleepless nights, I finally succeeded :-)
A complete solution, docker-compose based, including redis and postgres can be found at my github
https://github.com/tribp/netbox

ps: see also short story on medium:
https://petertribout.medium.com/netbox-installation-on-rpi-7634f9092268

greetings and succes
Peter
